src/annotator/config/settings.js

```diff
@@ -0,0 +1,107 @@
+'use strict';
+
+/**
+ * Return the href URL of the annotator `<link>` in the given document.
+ *
+ * This URL is used as the src of the sidebar's `<iframe>`.
+ *
+ * Return the value of the href attribute of the first
+ * `<link type="application/annotator+html" href="...">` element in the given
+ * document, or `null`.
+ *
+ * @param {Document} - The document to search.
+ * @return {string|null} - The URL to use for the sidebar's iframe.
+ *
+ */
+function iFrameSrc(document_) {
```

Comment:
I decided not to call this function `app()` :) But it is the function that returns the value of the `config.app` setting. In the future I'd like to rename `config.app` to `config.iFrameSrc` or else get rid of the setting entirely.